### DISCUSSION TITLE
Case Fatality Rate Delay

### DISCUSSION THREAD

**Discussion by valeance** [owid/covid-19-data#2573] [General]
Why is the Case Fatality Rate calculation delayed by 10 days? Would something higher be more appropriate for Omicron?  
![image](https://user-images.githubusercontent.com/43921778/159151989-0428e106-b6e9-46c7-addd-3007b17fdf3b.png)

This is what the case fatality curve would look like at different lag's between symptoms and death. 
![image](https://user-images.githubusercontent.com/43921778/159151942-4d902660-66ee-4b3c-ba99-bbfc562e1e8a.png)

**Comment by edomt** [comment]
Hi @valeance 

Are you aware of specific sources that indicate that the delay has become longer with omicron?

The choice of 10 days was made in late 2021, as the delay with the highest correlation between reported cases and reported deaths in the data published by Johns Hopkins University, between September 2020 and September 2021. We're not particularly attached to this number, but I personally haven't seen strong evidence that we should change it.

The resulting CFR seems quite robust if you zoom out and look at it over the course of the pandemic: 

<img width="944" alt="image" src="https://user-images.githubusercontent.com/7103774/160546985-1d861749-7560-4eec-a600-44637a6a56ba.png">

**Comment by valeance** [comment]
Sure. 
This is Delta-Wave. Notice how deaths peak almost exactly after cases peak. 
![image](https://user-images.githubusercontent.com/43921778/160655150-20620bd3-4969-4389-8a43-acfa790f36ee.png)

Vs. Omicron where deaths had a longer delay 
![image](https://user-images.githubusercontent.com/43921778/160655623-97d919ff-8a09-4733-9879-e6c3c0e83621.png)

Pretty clear there is a difference between the case-death curve.

**Reply by akburnham** [reply]
I'm new to this platform and did not see this discussion before posting my similar concern and suggestion yesterday that the delay should be 21 days in the US.  That flattens the case death rate curve.  I wrote a "technical light" article on this delay two years ago and concluded the delay was 16-21 days, depending on whether the delay was from infection or diagnosis.  https://www.researchgate.net/publication/341282145_What_do_Thunder_and_COVID-19_Statistics_Have_in_Common
Could reporting and medical care issues cause the delay to be country specific?
